packages/types/src/Delegation.ts

```diff
@@ -9,18 +9,19 @@ import type { ICType } from './CType'
 import type { IDidDetails } from './DidDetails'
 
 /* eslint-disable no-bitwise */
-export enum Permission {
-  ATTEST = 1 << 0, // 0001
-  DELEGATE = 1 << 1, // 0010
-}
+export const Permission = {
+  ATTEST: 1 << 0, // 0001
+  DELEGATE: 1 << 1, // 0010
+} as const
+export type Permissions = typeof Permission[keyof typeof Permission]
```

Comment:
Yes, `PermissionType` sounds also better, but I am afraid it might be confusing. What about renaming the `Permission` type to `PermissionType`, and the exported type `Permission`? So basically switching them around?